### PR TITLE
Add JSDoc and ReadMe Instructions

### PR DESCRIPTION
**Pull request by sarathkumar6**
* Added JSDoc to the methods in index.js
* Added missing method description in ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,16 +40,19 @@ schedule.addJob('<JOB_NAME>', options, async () => {
...
+
 #### Methods:
 | Methods       | Description                                     | Ready |
 | ------------- |:------------------------------------------------|:-----:|
-| addJob()      | Used to create a Job                            |   ✓   |
-| stopJob()     | Stops a Job but doesn't remove from the manager |   ✓   |
-| removeJob()   | Stops and Removes a Job from the Manager        |   ✗   |
+| addJob()      | Creates a job in the manager                    |   ✓   |
+| stopJob()     | Only stops a job in the manager                 |   ✓   |
+| removeJob()   | Stops & removes a job from the manager          |   ✓   |
```
Comment (AndreVarandas) [inline README.md]: Remove job is no longer available, was replaced by delete job.

File: src/index.js [hunk 2]
```diff
@@ -4,33 +4,56 @@ export class TeaTime {
   constructor() {
     this.jobs = new Map();
   }
-
+  /**
+   * @description Adds a job
```
Comment (AndreVarandas) [inline src/index.js:9]: Can you add a line break after each description just to keep the style? 😃

File: src/index.js [hunk 3]
```diff
@@ -4,33 +4,56 @@ export class TeaTime {
...
+   */
   addJob(name, time, method) {
     if (this.jobs.has(name) && 'action' in this.jobs.get(name)) {
       clearTimeout(this.jobs[name].action);
     }
 
     this.jobs.set(name, {
       time,
-      action: this.setJob(name, time, method),
+      action: this.setJob(name, time, method)
```
Comment (AndreVarandas) [inline src/index.js]: Please also keep trailing comma

File: src/utils/utils.js [hunk 4]
```diff
@@ -43,5 +45,5 @@ const getTime = (time) => {
 };
 
 export {
-  getTime,
+  getTime
```
Comment (AndreVarandas) [inline src/utils/utils.js]: we also want to keep this trailing comma

**Review by AndreVarandas — CHANGES_REQUESTED** [review]
Hi Thank you!
Please see my comments below 👍

**Review by AndreVarandas — APPROVED** [review]

**Comment by AndreVarandas** [comment]
Thank you for your work @sarathkumar6 ! 🚀

**Comment by sarathkumar6** [comment]
@AndreVarandas Thanks for the review and letting me to work on the issue

**Pull request merged**